src/wp-includes/class-wp-theme-json.php

```diff
@@ -523,6 +528,7 @@ public static function get_element_class_name( $element ) {
 		array( 'border', 'width' ),
 		array( 'color', 'link' ),
 		array( 'dimensions', 'minHeight' ),
+		array( 'position', 'sticky' ),
```

Comment:
Doesn't this also need to support position `fixed`? See https://github.com/WordPress/gutenberg/pull/46142/files#diff-b03597cc3da199e5aa5a94950e8241135904853e7c3b82d758e42744878afae7R553.

Reply:
Thanks for adding a comment on this one @felixarntz! I touched on this briefly in the PR description, but the appearance tools opt-in here intentionally deviates from what's in the Gutenberg plugin. Because `sticky` is the position support that is ready to be rolled out (and will be opted into for Group and Post Content blocks), in this core PR, it's added to the appearance tools opt-in, so that it's automatically exposed.

However, `fixed` is still a little experimental. In the Gutenberg plugin, it's good for it to be opted-into automatically so that it can be easily tested, however for the core merge, I think it's good for `fixed` to not be automatically opted-into at this stage.

Reply:
Thanks for clarifying @andrewserong, sorry I missed that 🤦

Reply:
No worries! It is a little unusual to deviate from the Gutenberg code, so thank you for double-checking 🙂